src/interfaces/IPlugin.sol

```diff
@@ -29,6 +24,14 @@ enum ManifestAssociatedFunctionType {
 }
 // forgefmt: disable-end
 
+struct ManifestExecutionFunction {
+    // TODO(erc6900 spec): These fields can be packed into a single word
+    // The selector to install
+    bytes4 executionSelector;
+    // If true, the function won't need runtime validaiton, and can be called by anyone.
+    bool isPublic;
```

Comment:
Perhaps `isUnprotected` or `isUnguarded` (or something else similar) is clearer?

Or maybe even `isRuntimeUnprotected`?

Reply:
Yeah good ideas. I also thought of `needsValidation`, but I thought the intuition around `isPublic` might be simpler.

Reply:
Using a name that includes `runtime` might help reduce user errors